frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -525,9 +530,6 @@
                 utilities.hideLoader();
             },
             onError: function(response) {
-                var error = response.data;
-                utilities.storeData('emailError', error.detail);
-                $state.go('web.permission-denied');
```

Comment:
This way, you have only hidden the error, but the API still didn't get called due to which challenge phase tab is empty.

Reply:
ok sir but if change it in api then how can i fix the test. like i dont know how is it being tested

Reply:
can you tell me the file which is testing it?